src/Directive/AbstractAdmonitionDirective.php

```diff
@@ -29,6 +29,10 @@ public function __construct(string $name, string $text)
 
     final public function processSub(Parser $parser, ?Node $document, string $variable, string $data, array $options): ?Node
     {
+        if (null === $document) {
+            throw new \RuntimeException('Content expected, none found.');
+        }
+
```

Comment:
A little extra thing - admonitions (notes, etc) MUST have content. For example, this is illegal:

```rst
.. note::

I forgot to indent a message on my note! I'm *note* having a good day ;)
```